### ISSUE TITLE
yolo face for yolo Nas !

### ISSUE THREAD

**Issue by oussaifi-majdi** [Deci-AI/super-gradients#925]
if i will use yolo Nas for face detection 
how can use it , I can't find Face Detector like other versions .

**Comment by BloodAxe** [comment]
The opened issue contains no description. 
Do you mind elaborating on your request? Otherwise it will be closed as invalid.

**Comment by oussaifi-majdi** [comment]
ah I'm sorry, my question is how can we use yolo Nas for face detection , I can't find Face Detector like other versions

**Comment by BloodAxe** [comment]
We provide pretrained weights for COCO which includes 80 classes including person. 
There is no specific class for face detection in that dataset. 
We hightly encourage our users to use this model to fine-tune to their specific needs.

Please refer to available notebooks and a documentation to learn more:

https://bit.ly/yolo-nas-starter-notebook
https://github.com/Deci-AI/super-gradients/blob/master/documentation/source/YoloNASQuickstart.md

**Comment by oussaifi-majdi** [comment]
What options should I use to run the training by using only the CPU?

**Comment by BloodAxe** [comment]
https://docs.deci.ai/super-gradients/documentation/source/device.html
